locallibrary/catalog/views.py

```diff
@@ -1,3 +1,24 @@
 from django.shortcuts import render
+from .models import Book, BookInstance, Author, Genre
 
-# Create your views here.
+
+def index(request):
+    num_books = Book.objects.all().count()
+    num_instances = BookInstance.objects.all().count()
+
+    # Available Books
+    num_instances_available = BookInstance.objects.filter(
+        status__exact="a"
```

Comment:
Tìm hiểu về enum để giúp code trở nên tường minh hơn em nhen, ko cần phải dùng comment chú thích cho các giá trị cứng như "a"
ví dụ 
```suggestion
        status__exact=BookStatus.AVAILABLE
```